### PR TITLE
chore: fix lint check

### PR DESCRIPTION
**Pull request by kangsorang**
# Description

Fix lint check (lycheeverse/lychee)

### CODE REVIEW

File: specs/meta/validator-deposit.md [hunk 1]
```diff
@@ -12,18 +12,20 @@
...
+The finalization of the output is checked when the next outputs are submitted, or if the finalization period
+of the submitted output has passed, you can directly
+[trigger unbond by using the `unbond` command](#try-unbond-in-validatorpool).
+For more details about submitting an output as a validator, see [here](../protocol/validator.md).
 
 This guide teaches you how to deposit, withdraw, or try to unbond in `ValidatorPool` via CLI. You can find the proxy
-address of `ValidatorPool` on Sepolia [here](../../packages/contracts/deployments/sepolia/ValidatorPoolProxy.json),
+address of `ValidatorPool` on Sepolia
+[here](https://github.com/kroma-network/kroma/blob/main/packages/contracts/deployments/sepolia/ValidatorPoolProxy.json),
 on Mainnet TBD.
```
Comment (0xHansLee) [inline specs/meta/validator-deposit.md]: Please update this as well.

File: specs/protocol/differences-from-optimism.md [hunk 2]
```diff
@@ -29,9 +29,9 @@
 
 There are two types of network participants in the OP Stack:
 
-- [Sequencers](https://github.com/ethereum-optimism/optimism/blob/develop/specs/introduction.md#sequencers) consolidate
+- [Sequencers](https://specs.optimism.io/introduction.html#sequencers) consolidate
```
Comment (0xHansLee) [inline specs/protocol/differences-from-optimism.md:32]:
```suggestion
- [Sequencers](https://specs.kroma.network/introduction.html#sequencers) consolidate
```

File: specs/protocol/differences-from-optimism.md [hunk 3]
```diff
@@ -29,9 +29,9 @@
...
  users' on/off chain transactions into blocks. They submit checkpoint outputs as well as batch transactions.
-- [Verifiers](https://github.com/ethereum-optimism/optimism/blob/develop/specs/introduction.md#verifiers) verify rollup
+- [Verifiers](https://specs.optimism.io/introduction.html#verifiers) verify rollup
```
Comment (0xHansLee) [inline specs/protocol/differences-from-optimism.md:34]:
```suggestion
- [Validators](https://specs.kroma.network/introduction.html#validators) verify rollup
```

File: specs/protocol/differences-from-optimism.md (hunk 2 above)
Comment (kangsorang) [reply specs/protocol/differences-from-optimism.md:32]: Since it's a reference to the OP stack, wouldn't the optimism link be correct?

**Review by 0xHansLee — APPROVED** [review]
TY

**Review by Pangssu — APPROVED** [review]

**Review by sm-stack — APPROVED** [review]

**Pull request merged**